### ISSUE TITLE
[ENH]: callable color map of Plasma with Deuteranopia colourblind filter

### ISSUE THREAD

**Issue by hrwakeford** [matplotlib/matplotlib#28148]
### Problem

It would be super useful if there was a cmap that was in the default colour-blind colours of the Plasma map. 

Image: left side color mask using Sim Daltonism application on Mac set to Deuteranopia colour-blindness, right side Plasma
![Screenshot 2024-04-29 at 10 45 02](https://github.com/matplotlib/matplotlib/assets/7988823/d102b556-6c84-4177-a49b-8a54c35df569)

This would mean that a colour-blind map is the same for a majority of readers and in the caption we can point to the same colours as what the majority Color-blind would see. This is especially pertinent in scientific publications but almost certainly has other uses too. 

### Proposed solution

Generate a color map that matches the gradients of the current maps when using Sim Daltonism or other software to generate Color-blind schemes that would look the same to "all" (barring the rarer types of Color-blindness) users.

**Comment by mwaskom** [comment]
It sounds like you might want to use the cividis map: https://github.com/benmaier/cividis.

> in the caption we can point to the same colours as what the majority

With any of the perceptually-uniform sequential colormaps in the viridis family, you should be able to use the universally-accessible terms "ligher" and "darker" to describe regions (or the polarity) of the colormap, and I'd suggest doing this over using hue terms like "blue" and "yellow". A problem with the colormap you're proposing here (and with cividis imo, but somewhat less so) is that it effectively turns the colormap into a bipolar scale that implies a categorical boundary somewhere around the midpoint, which may not be an accurate representation of the nature of your data. Unfortunately, color is super complicated.

**Comment by rcomer** [comment]
Note that cividis is one of Matplotlib's built in color maps, so you do not need to install the linked package.

**Comment by scottshambaugh** [comment]
Linking this related ticket https://github.com/matplotlib/matplotlib/issues/17893

**Comment by rcomer** [comment]
I am going to close this one as it seems like cividis should fulfil the requirements of the use-case, and because Matplotlib is in general very conservative about adding new colour maps.  It can always be re-opened if anyone thinks there is more to do here.
